node/node.d.ts

```diff
@@ -504,7 +504,8 @@ interface NodeBuffer extends Uint8Array {
     writeDoubleBE(value: number, offset: number, noAssert?: boolean): number;
     fill(value: any, offset?: number, end?: number): this;
     // TODO: encoding param
```

Comment:
Can you remove this comment as well. This change seems to satisfy the `// TODO`.